Spine/Operation.swift

```diff
@@ -211,12 +211,14 @@ class SaveOperation: Operation {
 		Spine.logInfo(.Spine, "Saving resource \(resource) using URL: \(request.URL)")
 		
 		HTTPClient.request(request.method, URL: request.URL, payload: request.payload) { statusCode, responseData, networkError in
-			if let networkError = networkError {
-				self.result = Failable(networkError)
-				self.state = .Finished
-				return
-			}
-			
+
+            if 400 ... 599 ~= statusCode! {
+                let error = NSError(domain: "networkError", code: statusCode!, userInfo: nil)
+                self.result = Failable(error)
+                self.state = .Finished
+                return
+            }
```

Comment:
This is because this projects uses tabs for indentation, and you used spaces. Apparently the tab size on GitHub is different from that in Xcode. You can change this in the Xcode preferences under *Text Editing > Indentation*.